frontend/src/map-screen/item-marker.js

```diff
@@ -32,7 +32,7 @@ class ItemMarker extends React.Component {
                     position={this.props.location}
                 >
                 { this.state.isOpen && this.state.activeMarker ?
-                    <InfoWindow maxWidth={400} defaultPosition={ this.props.location } onCloseClick={this.props.onToggleOpen}>
+                    <InfoWindow maxWidth={800} maxHeigth={800} defaultPosition={ this.props.location } onCloseClick={this.props.onToggleOpen}>
```

Comment:
Minor typo on line 35. Maybe from `maxHeigth={800}` to `maxHeight={800}`? Hehe! 😄